gateway-service/pom.xml

```diff
@@ -49,6 +49,18 @@
             <version>1.0-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.java.mentorship</groupId>
+            <artifactId>budget-contracts</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
Two things here:
The contract version can be extracted in the parent pom
The scope is compile by default, so you can remove it